CONTRIBUTING.md

```diff
@@ -6,9 +6,9 @@ If you want to contribute to the repository, here's a quick guide:
   2. Clone the repository into a local directory.  
 
   3. Install `Ruby`. Supported versions in the core are:
-     - 2.3.7
-     - 2.4.5
-     - 2.5.3
+     - 2.7
+     - 3.0
+     - 3.1
```

Comment:
I added one more change here, to update the supported rubies.  2.3, 2.4 and 2.5 are long since end of life.  Even 2.7 was recently end of life but could be the last 2.x version we support for now.  3.0 introduced kwargs and positional hash arguments as distinct things and could be problematic for old code until we get more testing with ruby 3.0 under our belts.